search_events_app/services/filters/features/repeating_events_filter.py

```diff
@@ -2,7 +2,7 @@
 
 
 class RepeatingEventsFilter(Filter):
-	CODE = "RE"
+	CODE = 'RE'
```

Comment:
you should add this in another as constants, and import wherever you need them. and this attribute should be in lower case

Reply:
Done!